src/chaplin/views/layout.coffee

```diff
@@ -97,7 +97,7 @@ module.exports = class Layout extends View
   openLink: (event) =>
     return if utils.modifierKeyPressed(event)
 
-    el = if Backbone.$ then event.currentTarget else event.delegateTarget
+    el = event.currentTarget
```

Comment:
Might need to revert this commit. It's the cause of 8 failling tests with NativeView.